lib/src/opentdf.ts

```diff
@@ -529,7 +529,7 @@ class NanoTDFReader {
       dpopEnabled: this.outer.dpopEnabled,
       dpopKeys: this.outer.dpopKeys,
       kasEndpoint: this.opts.allowedKASEndpoints?.[0] || 'https://disallow.all.invalid',
-      platformUrl: this.outer.platformUrl,
+      platformUrl: this.opts.platformUrl || this.outer.platformUrl,
```

Comment:
If we make it required in the constructor, won't this always resolve to the opts' field

Reply:
yes, it was mainly because this.opts.platform's type was string | undefined and platformUrl wanted just string. wasnt sure the best way around that in ts. what would you suggest?